package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "resin-typescript-skeleton",
+    "version": "0.0.1",
+    "description": "Skeleton template for a Resin TypeScript project",
+    "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+    "types": "build/index.d.ts",
```

Comment:
@pimterry Thoughts on this, please. It's strongly against the way MS recommend you do this, `types` should point to a declaration file and not a source file:

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html
https://github.com/Microsoft/TypeScript/issues/8305